docs/OV_Runtime_UG/migration_ov_2_0/intro.md

```diff
@@ -11,66 +11,68 @@
    openvino_2_0_configure_devices
    openvino_2_0_preprocessing
    openvino_2_0_model_creation
-      
+
 @endsphinxdirective
 
 ### Introduction
 
 Older versions of OpenVINO™ (prior to 2022.1) required to change the logic of applications when an user migrates from the frameworks like TensorFlow, ONNX Runtime, PyTorch, PaddlePaddle, etc. The change of application's logic is connected with:
 
-- Model Optimizer changed input precisions for some inputs. For example, neural langauge processing models with `I64` input are becoming to have `I32` input element type.
+- Model Optimizer changed input precisions for some inputs. For example, neural language processing models with `I64` input are becoming to have `I32` input element type.
 - Model Optimizer changed layouts for TensorFlow models (see [Layouts in OpenVINO](../layout_overview.md)). It leads to unexpected user behavior that a user needs to use a different layout for its input data with compare to the framework:
 ![tf_openvino]
 - Inference Engine API (`InferenceEngine::CNNNetwork`) also applied some conversion rules for input and output precisions because of device plugins limitations.
 - Users need to specify input shapes during model conversions in Model Optimizer and work with static shapes in the application.
 
-OpenVINO API 2.0 is introduced to align logic of working with model as it is done in the frameworks - no layout and precision changes, operates with tensor names and indeces to address inputs and outputs. OpenVINO Runtime is composed of Inference Engine API used for inference and ngraph API targeted to work with models, operations. The API 2.0 has common structure, naming convention styles, namespaces, removes duplicated structures. See [How to migrate to API 2.0](./common_inference_pipeline.md) for details.
+OpenVINO™ introduces API 2.0 to align logic of working with model as it is done in the frameworks - no layout and precision changes, operates with tensor names and indices to address inputs and outputs. OpenVINO Runtime is composed of Inference Engine API used for inference and nGraph API targeted to work with models, operations. The API 2.0 has common structure, naming convention styles, namespaces, removes duplicated structures. See [How to migrate to OpenVINO API v2](common_inference_pipeline.md) for details.
```

Comment:
why you changed to v2?? In other places of the document we have 2.0

CC @xu-yuan1 what is the proper name?

Reply:
The approved names by marketing are "API 2.0", "OpenVINO API v2" and "OpenVINO 2022.1". I think "OpenVINO API 2.0" is in a grey area as it's not forbidden explicitly. @andrew-zaytsev  Andrey may know better and confirm.
But I think on the same page we should keep consistent, having both "2.0" and "v2" can be confusing. Or maybe we can have a statement at the beginning to say that they are the same thing.